pyproject.toml

```diff
@@ -13,18 +13,22 @@ exclude = [
 python = "^3.7.1"
 pandas = "^1.3.5"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 flake8 = "^5.0.4"
 isort = "^5.11.5"
 pytest = "^7.2.1"
+pylint = "*"
+black = "*"
+
```

Comment:
Could we remove this whitespace

Reply:
Removed =) Sorry was testing something so made a blank line so I could push the code up =)